lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
+        company_name=company_name,
+        honor_code_html=cert_info_honor_code_html,
+        verifying_html=cert_info_verified_html
+    )
+
+    # Translators:
+    context['certificate_type_title'] = _('{certificate_type} Certfificate').format(
+        certificate_type=context.get('certificate_type')
+    )
+
+    # Translators:
+    context['certificate_verify_title'] = _("How {company_name} Validates Student Certificates").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['certificate_verify_description'] = _("Certificates issued by {company_name} are signed by a gpg key so "
+                                                  "that they can be validated independently by anyone with the "
+                                                  "{company_name} public key. For independent verification, "
+                                                  "{company_name} uses what is called a "
+                                                  "&quot;detached signature&quot;.").format(company_name=company_name)
+
+    # Translators:
+    context['certificate_verify_urltext'] = _("Validate this certificate for yourself")
+
+    # Translators:
+    context['company_about_description'] = _("{company_name} offers interactive online classes and MOOCs from the "
+                                             "world's best universities, including MIT, Harvard, Berkeley, University "
+                                             "of Texas, and many others.  {company_name} is a non-profit online "
+                                             "initiative created by founding partners Harvard and MIT.").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['company_about_title'] = _("About {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_about_urltext'] = _("Learn more about {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_courselist_urltext'] = _("Learn with {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_careers_urltext'] = _("Work at {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_contact_urltext'] = _("Contact {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_privacy_urltext'] = _("Privacy Policy")
+
+    # Translators:
+    context['company_tos_urltext'] = _("Terms of Service &amp; Honor Code")
+
+    # Translators:
+    context['document_banner'] = _("{company_name} acknowledges the following student accomplishment").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['logo_subtitle'] = _("Certificate Validation")
+
+    if certificate.mode == 'honor':
+        # Translators:
+        context['certificate_type_description'] = _("An {cert_type} Certificate signifies that an {company_name} "
```

Comment:
Oy.... this isn't always going to work, right? "An Verified Certificate"? "An Boeing learner"? Can we adjust this text, perhaps, to not use articles so explicitly?